daniellamarr/merry-sims-test
# #chore_number: server setup

[](### What does this PR do?
Set up server and add the dependencies necessary

## Description of task to be completed

- Sets up server to run on port 9090
- Configures babel for trans piling ECMA Script higher than ES5
- Adds a .env file and its dependency
- Adds a gitignore file
- Adds a start script to the application

## How should this be manually tested

1. Clone the repo and checkout to devlope then `npm i` to install dependencies
2. Start the app after cloning with `npm run start:dev`
2. The app should run on port 9090
3. Start the server on postman `http://localhost:9090`
4. Test linking files using `import`

## What are the relevant Stories
N/A

## Screenshots
N/A

## Questions:
N/A)
